Formula/ooniprobe.rb

```diff
@@ -200,41 +200,41 @@ def plist; <<-EOS.undent
    <dict>
      <key>Label</key>
        <string>#{plist_name}</string>
-     <key>Program</key>
-       <string>#{opt_bin}/ooniprobe</string>
+     <key>EnvironmentVariables</key>
+     <dict>
+       <key>PATH</key>
+       <string>#{HOMEBREW_PREFIX}/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
```

Comment:
On my version of macOS (Sierra) it appears that the PATH does not include the `HOMEBREW_PREFIX` path when starting programs via launchd. This is to ensure the HOMEBREW_PREFIX is present in there.